### ISSUE TITLE
StopSpam: account selection do not accessible for screen readers

### ISSUE THREAD

**Issue by dartraiden** [miranda-ng/miranda-ng#1059]
![2017-12-09_20h35_34](https://user-images.githubusercontent.com/5046855/33797886-8b405070-dd20-11e7-9a50-0417833b3036.png)

This is not standard checkboxes.
